CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Added functions `edge.GetLbAppRules`, `edge.GetLbServerPools`, `edge.GetLbAppProfiles`, `edge.GetNsxvNatRules`, `client.GetOrgList`
 * Exported private function `client.maxSupportedVersion` to `client.MaxSupportedVersion`
 * Able to upload an OVF without ovf:size defined in File part. Some bug fix for uploading OVA/OVF. [#331](https://github.com/vmware/go-vcloud-director/pull/331)
+* Add support for handling catalog storage profiles (`adminOrg.CreateCatalogWithStorageProfile`,
```

Comment:
I thought that catalog can only have ONE storage profile.

Reply:
I had this as a general term "storage profiles" and yes - it is true that it only handles one. Will change.

Reply:
Changed.